### ISSUE TITLE
diesel-cli ignores "--migration-dir" option

### ISSUE THREAD

**Issue by ensc** [diesel-rs/diesel#2225]
### Versions

- **Rust:** rustc 1.39.0 (4560ea788 2019-11-04)
- **Diesel:** 1.4.0
- **Database:** sqlite3 + postgres
- **Operating System** linux

### Feature Flags

- **diesel:**  sqlite xor postgres

## Problem Description

The `diesel database` command ignores the migration dir supplied by `--migration-dir` and always assumes `migrations`.

### What are you trying to accomplish?

```
$ diesel database --migration-dir='.migrations/sqlite' --database-url='migrate.sqlite3' reset
Unable to find migrations directory in this directory or any parent directories.

$ ls -l .migrations/
total 8
drwxrwxr-x. 6 ensc ensc 4096 Nov 21 12:49 sqlite
```

### Steps to reproduce

1. `mkdir test && cd test`

2. `mkdir sql`

3. ` diesel migration --migration-dir sql generate init`

4.  `diesel database --migration-dir sql setup`

**Comment by weiznich** [comment]
Ok this seems to be some kind of duplicate of #1301, but now for the `database` subcommand. I assume the same [fix as back then](https://github.com/diesel-rs/diesel/commit/0134f1527d8a62de4f89b0e604092dd262d135d2) should also work here.

As workaround: Just pass the `--migration-dir` flag after the last subcommand (setup in your case).

**Comment by Ten0** [comment]
Isn't this closed by #2230 ?

**Comment by weiznich** [comment]
No that's another issue, but the fix is quite similar, just for another command.

**Comment by IamSaquib** [comment]
If this hasn't been resolved, I'd like to start on this.

**Comment by weiznich** [comment]
Closed by #2299
